inc/TRestGeant4AnalysisProcess.h

```diff
@@ -80,8 +80,9 @@ class TRestGeant4AnalysisProcess : public TRestEventProcess {
     Bool_t fPerProcessSensitiveEnergy = false;
     Bool_t fPerProcessSensitiveEnergyNorm = false;
 
-    TVector3 fPrismCenter;
-    TVector3 fPrismSize;
+    // vectors for size and dimensions of prism used to compute "primaryOriginDistanceToPrism"
```

Comment:
This does not follow the Doxygen documentation scheme. Just as previous data members documentation